### ISSUE TITLE
Can null be supported when returning the results?

### ISSUE THREAD

**Issue by skibster** [dfilatov/jspath#65]
When null appears in the JSON, JSPath seems to ignore it. For example:

```
var data = JSON.parse('{"Colors":{"Color":[{"ColorName":"Cyan","ColorSpecifications":{"ColorSpec":[{"Name":"Printing Method","Value":"offset"}]}},{"ColorName":"Magenta","ColorSpecifications":{"ColorSpec":[{"Name":"Printing Method","Value":"offset"}]}},{"ColorName":"Yellow","ColorSpecifications":{"ColorSpec":[{"Name":"Printing Method","Value":null}]}},{"ColorName":"Black","ColorSpecifications":{"ColorSpec":[{"Name":"Printing Method","Value":"offset"}]}}]}}');

JSPath.apply('.Colors.Color.ColorSpecifications.ColorSpec', data);
```

Results in:

```
[
  {Name: "Printing Method", Value: "offset"},
  {Name: "Printing Method", Value: "offset"},
  {Name: "Printing Method", Value: null},
  {Name: "Printing Method", Value: "offset"}
]
```
But when asking down to the **Value**, I get:

```
JSPath.apply('.Colors.Color.ColorSpecifications.ColorSpec.Value', data);
```

```
[
  "offset",
  "offset",
  "offset"
]
```

But the correct result should be:

[
  "offset",
  "offset",
  null,
  "offset"
]

**Comment by dfilatov** [comment]
Looks reasonable, but it will broke backward compatibility (

**Comment by dfilatov** [comment]
Fixed in 0.4.0

**Comment by skibster** [comment]
Awesome! Thanks so much.
